jenkins-pipelines/artifacts-oel81.jenkinsfile

```diff
@@ -0,0 +1,15 @@
+#! groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+artifactsPipeline(
+    test_config: 'test-cases/artifacts/oel81.yaml',
+    backend: 'aws',
+    region_name: 'eu-west-1',
+    provision_type: 'spot_low_price',
```

Comment:
This parameter had been assigned to 'spot' in test-cases/artifacts/oel81.yaml

Reply:
but seem like in the pipeline we have some leftovers, that can be removed